eslint.config.mjs

```diff
@@ -84,6 +83,22 @@ export default tseslint.config([
       ],
       'no-restricted-syntax': ['error', ...NO_RESTRICTED_SYNTAX],
       'object-shorthand': 'error',
+      'prefer-const': ['error', { destructuring: 'all' }],
```

Comment:
This is the only real change. Everything else here was just alphabetization. Happy to move to another PR if that's preferred.